internal/config/config.go

```diff
@@ -394,13 +397,16 @@ func peerFromCR(p metallbv1beta2.BGPPeer, passwordSecrets map[string]corev1.Secr
 		return nil, err
 	}
 	keepaliveTime := p.Spec.KeepaliveTime.Duration
+	// DISCUSS3: When the Keepalive timer is set to zero, it indicates that the
+	// BGP speaker (router) will not send Keepalive messages to its peer. Should
+	// we allow it? it might makes sense for BFD?
 	if keepaliveTime == 0 {
 		keepaliveTime = holdTime / 3
 	}
 
-	// keepalive must be lower than holdtime
-	if keepaliveTime > holdTime {
-		return nil, fmt.Errorf("invalid keepaliveTime %q must be smaller than holdtime %q",
+	// keepalive must be lower than 1/3 holdtime, otherwise FRR adapts it.
```

Comment:
I think this code is shared between the modes, so relying on FRR to adapt it isn't always the case